app/models/SubscriptionDetails.scala

```diff
@@ -26,7 +26,9 @@ import uk.gov.hmrc.crypto.{Decrypter, Encrypter}
 
 final case class SubscriptionDetails(subscriptionResponse: SubscriptionResponse,
                                      subscriptionRequest: SubscriptionRequest,
-                                     registrationType: RegistrationType)
+                                     registrationType: RegistrationType,
+                                     businessType: Option[BusinessType]
+                                    )
```

Comment:
No need to put the bracket in a new line.

Reply:
Moved bracket up one line.